static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -89,7 +89,10 @@ class ProjectPerformance extends AsyncView<Props, State> {
         name: 'metric',
         type: 'select',
         label: t('Metric'),
-        choices: () => ['duration', 'lcp'],
+        choices: [
```

Comment:
Why is the type of fields above set to `any[]` when the return type is `Field[]`?

Reply:
Hmm I think I just copied the code from another file, updating